modular/blocksyncer/blocksyncer_options.go

```diff
@@ -187,6 +187,8 @@ func (b *BlockSyncerModular) serve(ctx context.Context) {
 
 // enqueueNewBlocks enqueues new block heights onto the provided queue.
 func (b *BlockSyncerModular) enqueueNewBlocks(context context.Context, exportQueue types.HeightQueue, currHeight uint64) {
+	ticker := time.NewTicker(1500 * time.Millisecond)
```

Comment:
Suggest 1500ms -> 500ms, the block interval is around 2s, if we use 1.5ms, that means in some case, you should wait around 1s to get the latest block, it's too slow.

Reply:
This coroutine is to get the latest height to write to the queue, may write several blocks of height in one request, and then another coroutine to consume, consumption takes time, so we do not always need to get the latest height